### ISSUE TITLE
[Administration] Gestion des EA comme administrateur

### ISSUE THREAD

**Issue by npaille** [claroline/Distribution#6253]
| Q             | A
| ------------- | ---
| Bug report?   | non. Bloquant
| Version?      | 12.5.4

La gestion des EA comme administrateur de plateforme n'existe plus.  Impossible d'avoir une vision globale. 
Pb : pour gérer/retrouver mes données (archives), retrouver les EA créés par mon équipe et d'attribuer une suite numérique logique de code.
Peut-on récupérer cette fonctionnalité dans les "Outils" de l'administration?

![image](https://user-images.githubusercontent.com/33163723/66191429-96b99500-e68e-11e9-9d4d-ce0872ffeb46.png)

**Comment by Elorfin** [comment]
En fait la fonctionnalité a été déplacée dans l'outil Espaces d'activités du bureau.
La section Mes espaces d'activités gérés contient ce qui était visible depuis l'admin.

**Comment by npaille** [comment]
Non, j'ai 44 EA là et pas mes 300 et +. Il n'y a pas pas tout ce que est visible par un admin (peut-être petit bug d'affichage en fonction des rôles)

**Comment by Elorfin** [comment]
C'est bon j'ai compris. Il manque tous les espaces archivés (413 sur les 467 de la base).

**Comment by npaille** [comment]
Et quand mon assistant crée un EA, je ne le vois pas non plus.

**Comment by Elorfin** [comment]
Voilà déjà une 1ère partie pour la gestion des espaces archivés : https://github.com/claroline/Distribution/pull/6255.

> Et quand mon assistant crée un EA, je ne le vois pas non plus (même si pas archivé... puisque nouveau).
Il faut qu'il pense à m'abonner alors que par défaut j'étais abonnée à tout

Je pourrais avoir un lien pour tester et voir ce qui manque stp ?

**Comment by npaille** [comment]
Merci Axel
Voici l'autre cas:
https://ecampus.estp.fr/#/desktop/workspaces/open/elearn027/home/accueil

= un EA créé par un autre admin (mon collègue) que je ne retrouve pas à l'interrogation ni en affichage

**Comment by Elorfin** [comment]
Je vois bien `elearn027` dans Espaces d'activités > Espaces d'activités gérés (c'est le 1er de la liste).
Je le vois avec le compte de Laurent et le tien en mode usurpation. 

Par contre, si on parle du widget sur le bureau, en effet il ne ressort pas parce qu'il s'agit de la liste "Liste de mes espaces d'activités" qui filtrent sur les espaces d'activités dans lesquels l'utilisateur loggué est inscrit. Pour pouvoir l'avoir dans ton cas il faudrait utiliser la liste `Liste d'espaces d'activités` qui est plus générale et liste tous les espaces que l'utilisateur loggué peut voir.

Il faut que je rajoute les descriptions manquantes sur les listes pour clarifier les choses.
